ci/build_setup.sh

```diff
@@ -101,5 +104,7 @@ function cleanup() {
   rm -f "${ENVOY_CI_DIR}"/bazel-*
   rm -rf "${ENVOY_CI_DIR}"/bazel
   rm -rf "${ENVOY_CI_DIR}"/tools
+  rm -f "${ENVOY_SRCDIR}"/ci/prebuilt/BUILD
+  touch "${ENVOY_SRCDIR}"/ci/prebuilt/BUILD
```

Comment:
Ah OK this confused me, I thought the diff was deleting the file, but it's making it empty. Why do we need the empty BUILD file?